.github/workflows/gradle-pr-build.yml

```diff
@@ -21,6 +21,9 @@ jobs:
         # When changing the list of JDK versions, the build configuration has to be changed by a repository admin. See
         # https://github.com/pinterest/ktlint/pull/1787#issuecomment-1409074092
         jdk: [ 8, 11, 17, 19 ]
+        exclude: # windows with JDK8 are *really* flaky
+          - os: windows-latest
+            jdk: 8
```

Comment:
Disabled Java 8 tests on Windows like [Detekt did](https://github.com/detekt/detekt/blob/43f0796c54a179bfd0e0e94c7c79128771bbbf2a/.github/workflows/pre-merge.yaml#L25-L27), if so we need to change the status check in repo settings.

Reply:
@shashachu Can you reconfigure as described in https://github.com/pinterest/ktlint/pull/1787#issuecomment-1409074092?

Reply:
done. removed windows jdk 8